vm/adb/adb.go

```diff
@@ -263,8 +269,10 @@ func (inst *instance) repair() error {
 	// working, probably due to the introduction of seccomp. Therefore,
 	// we revert this to `adb shell reboot` in the meantime, until a more
 	// reliable solution can be sought out.
-	if _, err := inst.adb("shell", "reboot"); err != nil {
-		return err
+	if inst.reboot {
+		if _, err := inst.adb("shell", "reboot"); err != nil {
+			return err
+		}
```

Comment:
Don't we need to move SleepInterruptible+waitForSSH below into this branch as well? We already waited for ssh above.